test/_init_tests.js

```diff
@@ -33,6 +33,9 @@ import * as describes from '../testing/describes';
 // All exposed describes.
 global.describes = describes;
 
+// Increase the before/after each timeout since certain times they have timedout
+// during the normal 2000 allowance.
+const BEFORE_AFTER_TIMEOUT = 5000;
```

Comment:
Does this actually change the timeout of the `before`/`after` phase? I thought it just changed the timeout for the entire test (which includes any `before`/`after` steps).